linera-chain/src/certificate/hashed.rs

```diff
@@ -46,6 +52,34 @@ impl<T> Hashed<T> {
     pub fn into_inner(self) -> T {
         self.value
     }
+
+    pub fn lite(&self) -> LiteValue
+    where
+        T: Has<ChainId>,
+    {
+        LiteValue {
+            value_hash: self.hash,
+            chain_id: *Has::<ChainId>::get(&self.value),
```

Comment:
I believe this would simply be `self.value.chain_id()` with `T: HasChainId`

Reply:
Simplified in [Simplify call to extract ChainId](https://github.com/linera-io/linera-protocol/pull/2887/commits/07bb76a04bcb7bcedf50b3170701e5a31b33fbea)